components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/GraphBasedSequenceHandler.java

```diff
@@ -144,6 +158,90 @@ private void displayLongWait(AuthenticationContext context, HttpServletRequest r
         }
     }
 
+    /**
+     * Handles the prompt.
```

Comment:
comment += Make redirect to prompt handler (authentication endpoint) URL when this is initial prompt state.
Executes the respective event handler function when prompt is returning with user input.